Makefile

```diff
@@ -28,8 +28,8 @@ SYMBIFLOW_DEVICES = xc7a50t xc7a100t xc7a200t xc7z010 xc7z020
 QUICKLOGIC_ARCHIVE = quicklogic.tar.gz
 QUICKLOGIC_URL = https://storage.googleapis.com/symbiflow-arch-defs-install/quicklogic-arch-defs-63c3d8f9.tar.gz
 
-INTERCHANGE_BASE_URL = https://storage.googleapis.com/fpga-interchange-tests/artifacts/prod/foss-fpga-tools/fpga-interchange-tests/presubmit/143/20210715-090243
-INTERCHANGE_VERSION = b00b616
+INTERCHANGE_BASE_URL = https://storage.googleapis.com/fpga-interchange-tests/artifacts/prod/foss-fpga-tools/fpga-interchange-tests/continuous/44/20210909-090121
+INTERCHANGE_VERSION = 839e153
```

Comment:
Maybe `$(shell grep "litex-hub::nextpnr-fpga_interchange" conf/nextpnr/environment.yml | sed -e's/.*=//')`?

Reply:
This will need to be updated similarly to symbiflow, for which the latest package links are generated and pushed to GCP. the interchange version refers to the `fpga-interchange-tests` revision: https://github.com/SymbiFlow/fpga-interchange-tests/commit/839e153741fbea05316959fe83e3fb792384638c